tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_deployment_tuning.py

```
@@ -0,0 +1,408 @@
+                )
+
+        # --- Check 11: Exporter still responsive after load ---
+        if endpoint["bearer_auth"]:
+            bearer_token = ome_helpers.create_prometheus_k8s_bearer_token()
+        final_sample = ome_helpers.scrape_metrics_text_sample(
+            exporter_pod, bearer_token=bearer_token, max_bytes=1024  # gitleaks:allow
+        )
+        assert final_sample and len(final_sample) > 100, (
+            "Exporter became unresponsive after load test. "
+            f"Response length: {len(final_sample) if final_sample else 0}"
+        )
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

(PT018)